index.html

```diff
@@ -2,11 +2,122 @@
+                <section class="call-to-action-1">
+                    <a class="work-with-me" href="/contact.html">Work With Me</a>
+                </section>
+                <section class="call-to-action-2">
+                    <a class="view-portfolio" href="/portfolio.html">View Portfolio</a>
+                </section>
+            </section>
+        <!-- end of cta-s -->
+
+        </section>
+        <!-- end of hero message -->
+
+
+    </section>
+    <!-- end of hero section -->
+    
+    <footer>
+        <!-- social media icons -->
+        <section class="social-media-icons">
+            <a class="facebook-icon" target="blank" href="https://www.facebook.com/braddoxogilo/"><img class="facebook-logo" src="/assets/icons/Facebook Icon.svg" alt="Facebook Icon"></a>
+            <a class="twitter-icon" target="blank" href="https://twitter.com/BraddoxOgilo"><img class="twitter-logo" src="/assets/icons/Twitter Icon.svg" alt="Twitter Icon"></a>
+            <a class="instagram-icon" target="blank" href="https://www.instagram.com/braddoxogilo/"><img class="instagram-logo" src="/assets/icons/Instagram Icon.svg" alt="Instagram Icon"></a>
+            <a class="linkedin-icon" target="blank" href="https://www.linkedin.com/in/braddox-ogilo-77980b3b/"><img class="linkedin-logo" src="/assets/icons/LinkedIn-Icon.svg" alt="LinkedIn Icon"></a>
+        </section>
+        <!-- end of social media icons -->
+
+        <!-- newsletter -->
+        <section class="newsletter" >
+            <h3>great things</h3>
+            <h3 class="come">come to those who sign up</h3>
+            <p class="products" >(Product updates, Discounts and more)</p>
+
+                <!-- email submission -->
+                <section class="email-address-send">
+                    <input type="text" id="#" name="email" class="email" placeholder="Your Email Adress">
+                    <!-- <p class="email" >Your Email Adress</p> -->
+                    <p class="send" >Send</p>
+                </section>
+                <!-- end of email submission -->
+            <p class="copyright" >Copyright 2020 1990 designs</p>
+        </section>
+        <!-- end of newsletter -->
+
+        <section class="footer-nav">
+            <!-- what I do -->
+            <section class="what-i-do">
+                <a href="/what-I-do.html"><h4 class="what-title"> what i do</h4></a>
+                <section class="what-i-do-items">
+                    <p class="strategy">strategy</p>
+                    <p class="branding">branding</p>
+                    <p class="digital">digital</p>
+                </section>
+            </section>
+            <!-- end of what I do -->
+
+            <!-- work -->
+            <section class="portfolio">
+                <a href="/portfolio.html"><h4 class="portfolio-title">portfolio</h4></a>
+                <section class="portfolio-items">
+                    <p class="portfolio-strategy">all categories</p>
```

Comment:
put these as links in a ul
```suggestion
<ul>
<li>< a href="#">all categories</a></li>
<li></li>
</ul>
```

Reply:
Yeah, I think so too. Let me change that. Thanks for the feedback.

Reply:
This has been updated @DerKip